project/ws/app/src/lib/routes/app-toc/components/app-toc-home/app-toc-home.component.html

```diff
@@ -85,8 +85,7 @@
 <div class="flex flex-row gap-6 fixed-width">
   <div class="toc-content">
     <div class="pb-4 lg:py-4">
-      <ws-widget-content-toc [content]="content" [tocStructure]="tocStructure"   [forPreview]="false"
-      [resumeData]="resumeData" [batchData]="selectedBatchData" [skeletonLoader]="skeletonLoader" ></ws-widget-content-toc>
+      <ws-widget-content-toc [content]="content" [tocStructure]="tocStructure" [forPreview]="false" [resumeData]="resumeData" [batchData]="selectedBatchData" [skeletonLoader]="skeletonLoader" [changeTab]="changeTab"></ws-widget-content-toc>
```

Comment:
Please check this line of code showing 

 If 'ws-widget-content-toc' is an Angular component and it has 'initialRouteData' input, then verify that it is part of this module, please check